src/component/parser/json/converter/utils.ts

```diff
@@ -104,6 +104,14 @@ export class ConvertedElements {
   registerCategoryValue(id: string, value: string): void {
     this.categoryValues.set(id, { value });
   }
+
+  private callActivitiesCallingProcess: Map<string, ShapeBpmnElement> = new Map();
+  findCallActivityCallingProcessByProcess(id: string): ShapeBpmnElement {
+    return this.callActivitiesCallingProcess.get(id);
+  }
+  registerCallActivityCallingProcessByProcess(callActivity: ShapeBpmnElement, calledElement: string): void {
+    this.callActivitiesCallingProcess.set(calledElement, callActivity);
+  }
```

Comment:
⚠️ `callActivitiesCallingProcess` methods are only used in `ProcessConverter` as we already do for `elementsWithoutParentByProcessId`.
So we should move it directly there for a better encapsulation and to simplify future refactoring.

Reply:
Good catch 😉